src/_includes/graphql/customer-orders-output.md

```diff
@@ -211,6 +211,28 @@ Attribute | Data type | Description
 `name` | String | The name part of the name/value pair
 `value` | String | The value part of the name/value pair
 
+#### OrderAddress attributes {#OrderAddress}
+
+The `OrderAddress` object can contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`city` | String! | The city or town
+`company` | String | The customer's company
+`country_code` | CountryCodeEnum | The customer's country
+`fax` | String | The fax number
+`firstname` | String! | The first name of the person associated with the shipping/billing address
+`lastname` | String! | The family name of the person associated with the shipping/billing address
+`middlename` | String | The middle name of the person associated with the shipping/billing address
+`postcode` | String | The customer's ZIP or postal code
+`prefix` | String | An honorific, such as Dr., Mr., or Mrs.
+`region` | [CustomerAddressRegion](#customerAddressRegionOutput) | An object that defines the customer's state or province
+`region_id` | Int | The unique ID for a pre-defined region
```

Comment:
region_id is type ID as region_id: ID @doc(description: "The unique ID for a pre-defined region")